mantis-control-plane/mantis-control-plane-server/src/main/java/io/mantisrx/server/master/persistence/KeyValueBasedPersistenceProvider.java

```diff
@@ -130,19 +129,14 @@ public class KeyValueBasedPersistenceProvider implements IMantisPersistenceProvi
     private final LifecycleEventPublisher eventPublisher;
     private final Counter noWorkersFoundCounter;
     private final Counter workersFoundCounter;
+    private final MeterRegistry meterRegistry;
 
-    public KeyValueBasedPersistenceProvider(KeyValueStore kvStore, LifecycleEventPublisher eventPublisher) {
+    public KeyValueBasedPersistenceProvider(KeyValueStore kvStore, LifecycleEventPublisher eventPublisher, MeterRegistry meterRegistry) {
         this.kvStore = kvStore;
         this.eventPublisher = eventPublisher;
-        Metrics m = new Metrics.Builder()
-            .id("storage")
-            .addCounter("noWorkersFound")
-            .addCounter("workersFound")
-            .build();
-
-        m = MetricsRegistry.getInstance().registerAndGet(m);
-        this.noWorkersFoundCounter = m.getCounter("noWorkersFound");
-        this.workersFoundCounter = m.getCounter("workersFound");
+        this.meterRegistry = meterRegistry;
+        this.noWorkersFoundCounter = meterRegistry.counter("Storage_noWorkersFound");
```

Comment:
minor comment: pl keep `storage` in lower-case to be consistent with original metrics.
It'd be `storage` instead of `Storage`